__tests__/app.test.js

```diff
@@ -46,3 +46,39 @@ describe("/api", () => {
       });
   });
 });
+
+describe("/api/articles/:article_id", () => {
+  test("GET 200 Responds with an article object containing the correct properties", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article_id).toBe(1);
+        expect(body.title).toBe("Living in the shadow of a great man");
+        expect(body.topic).toBe("mitch");
+        expect(body.author).toBe("butter_bridge");
+        expect(body.body).toBe("I find this existence challenging");
+        expect(typeof body.created_at).toBe("string");
```

Comment:
you will have issues with testing against dates so this way of testing it is fine, but you could also do something like `expect(body.created_at).toEqual(expect.any(Date))`